var/spack/repos/builtin/packages/py-torch/package.py

```diff
@@ -171,7 +172,8 @@ class PyTorch(PythonPackage, CudaPackage):
     # https://github.com/pytorch/pytorch/pull/37865
     # Fixes CMake configuration error when XNNPACK is disabled
     patch('xnnpack.patch', when='@1.5.0')
-
+    #Fix rocm for py-torch 1.5 rel https://github.com/pytorch/pytorch/commit/1093e26d7288e3795da8163ad2516206d4ce6b63
+    patch('rocm.patch', when='@1.5.0+rocm')
```

Comment:
Will other versions of PyTorch build with ROCM, or do we need to backport this patch to older releases to get that working?